src/conversion/downcast-converters.js

```diff
@@ -45,17 +41,23 @@ import cloneDeep from '@ckeditor/ckeditor5-utils/src/lib/lodash/cloneDeep';
  *
  * 		downcastElementToElement( {
  * 			model: 'heading',
- * 			view: modelElement => new ViewContainerElement( 'h' + modelElement.getAttribute( 'level' ) )
+ * 			view: ( modelItem, consumable, conversionApi ) => {
+ * 				const viewWriter = conversionApi.writer;
+ *
+ * 				return viewWriter.createContainerElement( 'h' + modelElement.getAttribute( 'level' ) );
```

Comment:
`modelElement` -> `modelItem`

Reply:
Not that related, but there is an issue that we should try to unify parameters passed to callbacks in downcast and upcast. `( modelItem, viewWriter ) => { ... }` would look better.

Reply:
For sure, but it should be done in a follow-up.